galite-tests/src/test/kotlin/org/kopi/galite/tests/ui/vaadin/form/MultipleBlockFormTests.kt

```diff
@@ -238,6 +242,23 @@ class MultipleBlockFormTests : GaliteVUITestBase() {
     assertEquals(1, block.editor.item.record)
   }
 
+  @Ignore
+  @Test
+  fun `test activate record in multi block`() {
+    formExample.open()
+
+    val salesBlockModel = formExample.salesBlock.findBlock()
+
+    assertEquals(-1, salesBlockModel.activeRecord)
```

Comment:
you can use `salesBlockModel.model.activeRecord` instead of `salesBlockModel.activeRecord`. It is not clear why the activeRecord does get initiated with value 0 instead of -1